### ISSUE TITLE
Where are the complete labels and images

### ISSUE THREAD

**Issue by tasnimislam** [karstenBehrendt/boxy#7]
Hello, 
Nice job in labelling and collecting the images. After investigating I could only found 30k images those are properly annotated. If the labels exist the images are not there, if the images exist the labels are not there. If you can help on this.

**Comment by karstenBehrendt** [comment]
Hi @tasnimislam, sorry for the late response. I just went through our labels and images, and could not reproduce what you are describing.

Can you please provide additional information?

**Comment by yazeedhasan97** [comment]
Hi @tasnimislam @karstenBehrendt 

Which of the .zip files continues the 30K labeled images, please?
Sorry, but I don't have enough internet to download and check them all

**Comment by karstenBehrendt** [comment]
@yazeedhasan97 There are 200,000 annotated images as part of the complete dataset. About 170,000 are part of the training and validation sets which would leave around 30,000 images for the test set. If you mean those labels, they are hidden and only used for evaluating submissions.

The json files contain the labels. They are a couple hundred MB in size. 

Please re-open this task if this you feel that this does not address your problem.
